android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/INIParser.java

```diff
@@ -0,0 +1,123 @@
+package com.tavultesoft.kmea.util;
+
+import java.util.Map;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A custom, lightweight INI parser designed initially for use with Keyman's .kmp and .inf files.
```

Comment:
We should probably reference the kmp.inf spec. At the moment, it lives in a Google doc.
Maybe we should persist that on the keymanapp/keyman wiki where it's publicly visible?

Reply:
Gabe put it online at https://github.com/keymanapp/keyman/wiki/KMP-Metadata-File-(kmp.inf). That's now the master reference.

Reply:
Probably my biggest question/concern:  what are legal characters within individual elements of value lists?  What about within double-quoted elements?